platform/server-bundle/src/main/resources/webapp/js/services.js

```diff
@@ -87,6 +87,8 @@
                 dialog.getModalBody().css('padding', '0');
 
                 dialog.open();
+
+                dialog.getModal().prev().css('z-index', dialog.getModal().css('z-index'));
```

Comment:
This seems like a big hack, not a fix. What are we referring to with "prev()" here? I assume this will depend on the place it's called from? Why is the problem only appearing in some places?